### PR TITLE
[FEATURE] ViewHelper to check feature flags

### PR DESCRIPTION
**Pull request by linawolf**
Resolves https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/943 Releases: main

### CODE REVIEW

File: Documentation/Configuration/FeatureToggles.rst [hunk 1]
```diff
@@ -165,7 +165,22 @@ One can check whether a feature is enabled in TypoScript with the function
...
+    A new condition-based Fluid ViewHelper which allows
+    integrators to check for feature flags from within Fluid templates.
+
+The :ref:`t3viewhelper:typo3-fluid-feature` can be used to check for a feature in a Fluid 
+template:
+
+..  code-block:: html
+    :caption: EXT:myExtension/Resources/Private/Templates/SomeTemplate.html
+
+    <f:feature name="myFeatureFlag">
```
Comment (garvinhicking) [inline Documentation/Configuration/FeatureToggles.rst:183]:
```suggestion
    <f:feature name="unifiedPageTranslationHandling">
```

File: Documentation/Configuration/FeatureToggles.rst [hunk 2]
```diff
@@ -165,7 +165,22 @@ One can check whether a feature is enabled in TypoScript with the function
...
-    [feature("unifiedPageTranslationHandling") === false]
-        # This condition matches if the feature toggle "unifiedPageTranslationHandling" is false
-    [END]
+..  _feature-toggles-viewhelper:
+
+Feature toggles in Fluid
+========================
+
+..  versionadded:: 13.2
+    A new condition-based Fluid ViewHelper which allows
```
Comment (garvinhicking) [inline Documentation/Configuration/FeatureToggles.rst:174]: Is there a word missing like "was added" or maybe drop "which"?

**Review by garvinhicking — APPROVED** [review]
Thanks for the quick implementation. Just a minor note.

File: Documentation/Configuration/FeatureToggles.rst (hunk 2 above)
Comment (linawolf) [inline Documentation/Configuration/FeatureToggles.rst:174]:
```suggestion
    A new condition-based Fluid ViewHelper was added. It allows
```

**Review by garvinhicking — APPROVED** [review]

**Pull request merged**